### PR TITLE
Add ping command

### PR DESCRIPTION
**Pull request by pedrofracassi**
Can it get more obvious than this?

### CODE REVIEW

File: commands/ping.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  constructor(client) {
+    super(client);
+
+    this.name = "ping";
+  }
+
+  run(message) {
+    var embed = this.client.getDekuEmbed(message).setDescription(':ping_pong: `...`');
+    var startTime = Date.now();
+    message.channel.send({embed}).then((message) => {
```
Comment (davipatury) [inline commands/ping.js]: `.then(newMsg => {`

File: commands/ping.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        var endTime = Date.now();
+        embed.setDescription(`:ping_pong: \`${Math.round(endTime - startTime)}ms\``)
```
Comment (davipatury) [inline commands/ping.js]:
```js
`:ping_pong: \`${(newMsg.createdAt - message.createdAt)}ms\``
```

File: commands/ping.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  constructor(client) {
+    super(client);
+
+    this.name = "ping";
+  }
+
+  run(message) {
+    var embed = this.client.getDekuEmbed(message).setDescription(':ping_pong: `...`');
+    var startTime = Date.now();
+    message.channel.send({embed}).then((newMessage) => {
```
Comment (davipatury) [inline commands/ping.js]: `newMessage` aqui e `newMsg` lá embaixo?

File: commands/ping.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        var endTime = Date.now();
+        embed.setDescription(`:ping_pong: \`${(newMsg.createdAt - message.createdAt)}ms\``)
+        message.edit({embed});
```
Comment (davipatury) [inline commands/ping.js]: `newMsg.edit`

File: commands/ping.js (hunk 3 above)
Comment (pedrofracassi) [reply commands/ping.js]: sim

File: commands/ping.js (hunk 4 above)
Comment (pedrofracassi) [reply commands/ping.js]: fuk

File: commands/ping.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    this.name = "ping";
+  }
+
+  run(message) {
+    var embed = this.client.getDekuEmbed(message).setDescription(':ping_pong: `...`');
+    var startTime = Date.now();
+    message.channel.send({embed}).then((newMessage) => {
+        var endTime = Date.now();
+        embed.setDescription(`:ping_pong: \`${(newMessage.createdAt - message.createdAt)}ms\``)
+        message.edit({embed});
```
Comment (davipatury) [inline commands/ping.js]: newMessage

Comment (pedrofracassi) [reply commands/ping.js]: GRR

**Pull request merged**